sycl/doc/AMDGettingStartedAIE.md

```diff
@@ -6,61 +6,66 @@ change rapidly or get outdated.
-  * Recent Linux distribution
-  * Vitis 2022.2 which includes CHESS for AIE
 
-# Xilinx AIE Compilation
+- Recent Linux distribution;
+- Vitis 2022.2 which includes CHESS for AIE.
 
-This document aims to cover the key differences of compiling SYCL for Xilinx
+# AMD AIE Compilation
+
+This document aims to cover the key differences of compiling SYCL for AMD
 AIE using the WIP device compiler based on the upstream Intel SYCL compiler.
 
-The main future goal of this SYCL device compiler and SYCL runtime is to target 
+The main future goal of this SYCL device compiler and SYCL runtime is to target
 the AIE PL and APUs, not just the PS and AI Engine cores. The ideal goal is
 for all of the various AIE components to be usable seamlessly in a single
-source SYCL application. This however is a long term goal. For now we support 
-Xilinx FPGAs and AI Engine Cores separately and in an experimental WIP fashion.
+source SYCL application. This however is a long term goal. For now we support
+AMD FPGAs and AI Engine Cores separately and in an experimental WIP fashion.
 
 ## The SYCL Runtime for AIE
 
-The SYCL runtime for AIE is currently not the SYCL library compiled and 
-packaged with the device compiler in this project. The library packaged with 
-this project is currently a modified (for Xilinx FPGA) variation of the Intel 
-upstream implementation. It's currently only aimed at targeting Intel GPUs, 
-Intel FPGAs, Intel CPUs, Nvidia GPUs, AMD GPUs and Xilinx FPGAs.
+The SYCL runtime for AIE is currently not the SYCL library compiled and
+packaged with the device compiler in this project. The library packaged with
+this project is currently a modified (for AMD FPGA) variation of the Intel
+upstream implementation. It's currently only aimed at targeting Intel GPUs,
+Intel FPGAs, Intel CPUs, Nvidia GPUs, AMD GPUs and AMD FPGAs.
 
 There are 2 libraries using the same compiler:
- - ACAP++ see API in [test](../test/acap/test_aie_mandelbrot.cpp)
- - AIE++ see API in [test](../test/aie/mandelbrot.cpp)
-None of them are available for now. 
+
+- ACAP++, see API in [test](../test/acap/test_aie_mandelbrot.cpp);
+- AIE++, see API in [test](../test/aie/mandelbrot.cpp)
+
+which are available from [triSYCL](https://github.com/triSYCL/triSYCL).
 
 The ACAP++ library is usable as a standalone SYCL runtime library without the
-device compiler and defaults to a multi-threaded CPU application (all kernels are 
-executed on the host). This makes it superb for debugging your applications 
-before trying to target your application on a more complex heterogeneous 
+device compiler and defaults to a multi-threaded CPU application (all kernels are
+executed on the host). This makes it superb for debugging your applications
+before trying to target your application on a more complex heterogeneous
 architecture like AIE.
 
-The AIE++ library for now only targets AIE hardware.
+The AIE++ library for now only targets AIE hardware as the CPU
+implementation is a work-in-progress.
```

Comment:
we can already do quite many thing with AIE++ on cpu if I remember correctly.
it might be nice to show the difference in APIs between the two. since the better API is part of why AIE++ was built.

Reply:
Indeed. But since I have not documented it yet, my plan was to add this at the same time.
Nice to see you again in the area. ;-)